crates/sui-framework/sources/display.move

```diff
@@ -51,7 +51,7 @@ module sui::display {
     ///
     /// Uses only String type due to external-facing nature of the object,
     /// the property names have a priority over their types.
-    struct Display<phantom T: key> has key {
+    struct Display<phantom T: key> has key, store {
```

Comment:
A bit concerned about the `store`-- isn't `Display` is something that needs to be easy for clients to discover? I can definitely imagine someone wrapping a `Display` object + the API for finding it stops working as a result.

Or maybe this isn't a problem if the API looks at events instead of the object?

Reply:
It shouldn't be a problem if we rely on the `DisplayUpdated` event. Which I currently see as the main way of publishing / committing a state.

Reply:
We have 3 potential developments:

1. Make it key-only and allow single owner. That's the way to ignore access control + keep discoverability. At the same time everything is tied to an account (which doesn't seem like the best strategy).

2. Make it key and allow shared - we're forced to implement access control. Not good, too many cases, not as flexible as it could be.

3. Full freedom - store + key - use at your own risk. We can provide a generic primitive to lock and issue a cap for access (which I'm currently planning to do), or just don't share the object and keep it private - also works. Since we're making a bet on Collectible, this buddy can implement some of the security logic out of the box if Collectible is an all-in-one package.